plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMergeSink.java

```diff
@@ -209,31 +210,27 @@ private FileWriter createParquetFileWriter(String path, List<DeltaLakeColumnHand
         try {
             Closeable rollbackAction = () -> fileSystem.deleteFile(path);
 
-            List<Type> parquetTypes = dataColumns.stream()
-                    .map(column -> {
-                        Type type = column.getType();
+            List<TypedColumn> columns = dataColumns.stream()
+                    .map(dataColumn -> {
```

Comment:
Why did you consider renaming the variable name from `column` to `dataColumn` ?

Reply:
to avoid misunderstandings, all `List<TypedColumn>` inclusions renamed to `typedDataColumns`.